epochStart/shardchain/export_test.go

```diff
@@ -0,0 +1,6 @@
+package shardchain
+
+// SaveEpochStartInfoToStaticStorer -
+func (t *trigger) SaveEpochStartInfoToStaticStorer() error {
+	return t.saveEpochStartInfoToStaticStorer()
```

Comment:
although it is not used from a separate goroutine in tests, I think the mutex lock for this operation should be added as suggested also on the method's comment:
```
t.mutTrigger.Lock()
defer t.mutTrigger.Unlock()
```